internal/gpsd/objects.go

```diff
@@ -27,6 +28,23 @@ type TPV struct {
 	EPX, EPY, EPV       json.Number // Lat, Lon, Alt error estimate in meters, 95% confidence. Present if mode is 2 or 3 and DOPs can be calculated from the satellite view.
 	Track, Speed, Climb json.Number
 	EPD, EPS, EPC       json.Number
+	GridSquare          string `json:"-"`
+}
+
+func (t *TPV) CalculateGridSquare() error {
+	lat, err := t.Lat.Float64()
+	if err != nil {
+		return err
+	}
+
+	lon, err := t.Lon.Float64()
+	if err != nil {
+		return err
+	}
+
+	point := maidenhead.NewPoint(lat, lon)
+	t.GridSquare, err = point.GridSquare()
+	return err
```

Comment:
This is no longer needed I believe? `conn.GetGridSquare()` is not using the TPV's GridSquare, but calculates from LatLon on the fly.

Reply:
Hi Martin!

You are correct...it is an artifact that I forgot to remove.  Thanks!